doc/topics/releases/oxygen.rst

```diff
@@ -281,6 +281,54 @@ Salt Minion now no longer need to be configured against a specifig DNS name or I
 For this feature Salt Master now requires port 4520 for UDP broadcast packets to be opened
 and the Salt Minion be able to send UDP packets to the same port.
 
+Configuration
+=============
+
+By default, automatic discovery is disabled.
+
+Warning: due to the current limitations that will be changing in a future, before you turn on autodiscovery,
+make sure your network is secured and trusted. See the limitations below.
+
+Autodiscovery is configured on Master and Minion. Both of them are configured via ``discovery`` option
```

Comment:
For clarity, this should be "Autodiscovery is configured both on the Master and Minion".